metamusic/clients/songmetadata.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+"""
+Author : Salvador <Salvador@fedora>
+Date   : 2022-10-15
+Purpose: New program
+"""
+from metamusic.models import Track
+import argparse
+import eyed3
+
+
+# --------------------------------------------------
+def get_args():
+    """Get command-line arguments"""
+
+    parser = argparse.ArgumentParser(
+        description="eyeD3 client to get or update metadata",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "-t",
+        "--track",
+        help="A track file to get or update metadata",
+        metavar="str",
+        type=str,
+        required=True,
+    )
+
+    parser.add_argument(
+        "-i",
+        "--image",
+        help="An image file to update album artwork",
+        metavar="str",
+        type=str,
+        required=True,
+    )
+
+    return parser.parse_args()
```

Comment:
suggestion: Move this to the cli.py file and start creating a single argparse that handles all the cases you want.